src/server/server.jl

```diff
@@ -37,16 +40,14 @@ struct MonitoredOPAServer
     stopped::Ref{Bool}
     lck::ReentrantLock
 
-    function MonitoredOPAServer(configfile::String;
+    function MonitoredOPAServer(
+        cmdline::CommandLine,
```

Comment:
Bikeshedding a bit, but could maybe have a convenience outer constructor

```julia
MonitoredOPAServer(exec::Function, configfile::String; kwargs...)
= MonitoredOPAServer(CommandLine(exec), configfile; kwargs...)

# Use as:
MonitoredOPAServer(OpenPolicyAgent_jll.opa, ...)
```

Users generally care about the `MonitoredOPAServer` object, and not the `CommandLine` object, right?

Reply:
Done in https://github.com/JuliaComputing/OpenPolicyAgent.jl/pull/17/commits/96120546b8d970899c27f91f957c371ec3d554e6